### DISCUSSION TITLE
How to write new meta tags IPTC

### DISCUSSION THREAD

**Discussion by yarnik1** [ImageMagick/ImageMagick#2786] [Help] [answered]
How to write new meta tags Creator and Copyright for IPTC
I read the documentation but I can not understand how to do it.
I am interested in the copyright record for the site https://active-print.kiev.ua/
Please write an example.

**Comment by fmw42** [comment, marked as answer]
Imagemagick only reads IPTC data. It does not write to it. You can write a comment or a label but that is all.

**Comment by t246246** [comment]
FYI, there is neither `Creator` nor `Copyright` in IPTC, I guess it is `Credit` (or `By-line`) and `Copyright Notice` officially. See https://www.iptc.org/std/IIM/4.1/specification/IIMV4.1.pdf

Anyway you can do it exiftool or exiv2:

% exiv2 -M"add Iptc.Application2.Credit creator" -M"add IptcApplication2.Copyright copyright" yourfile.jpg

or

% exiftool -iptc:credit="creator" -iptc:copyrightnotice="copyright" yourfile.jpg
